src/tabs/AOCTab/AdventOfCodeTab.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SolutionsView from './SolutionsView'; 
+
+
+const AdventOfCodeTab = () => {
+  
+  return(    
+    <div>
+      <SolutionsView />
+    </div>
+  );
+}
+
+export default AdventOfCodeTab;
```

Comment:
We want to keep the convention of having an empty line at the end of the file. Look at what the red "-" says when you hover over it. Do this for all the files that have the same thing